package.json

```diff
@@ -99,12 +99,12 @@
           "default": ""
         },
         "coder.defaultUrl": {
-          "markdownDescription": "This will be shown in the URL prompt, along with CODER_URL if set, for the user to select when logging in. If autologin is enabled, this URL will be automatically logged into when the extension is activated.",
+          "markdownDescription": "This will be shown in the URL prompt, along with CODER_URL if set, for the user to select when logging in.",
```

Comment:
This line, I would change CODER_URL to "the default url" or something.

Reply:
My worry with that is there is nowhere else that describes the `CODER_URL` environment variable will be read and used in the URL prompt, but maybe it is superfluous to this setting, technically.  :thinking:

Reply:
OH WAIT I get it now in my head this setting was describing the URL prompt not just the default URL yeah it's perfect as is.